app/models/invenio_rdm_record_converter.rb

```diff
@@ -176,7 +175,9 @@ def invenio_metadata
         "additional_descriptions":
           format_additional("description", "acknowledgements", @generic_file.acknowledgments) +
           format_additional("description", "abstract", @generic_file.abstract) +
-          format_additional("description", "other", @generic_file.based_near, "presentation_location: "),
+          format_additional("description", "other", @generic_file.based_near, "presentation_location: ") +
+          [{"description":  "number_in_sequence: #{@generic_file.page_number.to_s.force_encoding("UTF-8")}", "type": {"id": "other"}}] +
```

Comment:
Weirdly, `page_number` is a string and not an array. It seems that `page_number_actual` is stored as an integer and used for Solr queries and sorting. `page_number_actual`, from what I saw in the specs, will only be present if `page_number`  does not have any characters in it. E.g something like "29a" will result in a blank `page_number_actual` , but I think this is just to prevent errors with Solr.